README.md

```diff
@@ -6,6 +6,8 @@
 
 [![CLOMonitor](https://img.shields.io/endpoint?url=https://clomonitor.io/api/projects/cncf/cloudevents/badge)](https://clomonitor.io/projects/cncf/cloudevents)
 
+document language: [English](README.md) | [简体中文](README.zh-cn.md)
```

Comment:
can you make it just "Language" instead?

Reply:
yes  Please wait me

Reply:
I'm a little nervous about this change. While on the surface it looks nice, it means each time a new language is added we need to touch ALL documents that have this line. I wonder if it might be better to have a more generic line in each doc that says something like "Other languages available" and directs them to a doc in the spec/languages directory that lists all of the languages available - each with a pointer to that language's top-level README.

What do other people think?